### DISCUSSION TITLE
Discord Invitation Issues

### DISCUSSION THREAD

**Discussion by PDL-II** [wintercms/winter#212] [Q&A] [answered]
Hi there, 

I am Octobercms user trying to migrate to Wintercms as well as join the Discord. But when I got the phone verification and inserted it, I got an error message and haven't been able to join. Can anyone help? 

Thanks in advance. :)

**Comment by mjauvin** [comment]
How can we help? Did you try multiple times? Maybe that was a temporary glitch? 

Try this invite:
https://discord.gg/djt6grJE

**Comment by bennothommo** [comment, marked as answer]
@PDL-II you will need to get in touch with [Discord Support](https://support.discord.com/hc/en-us) if you're having issues with phone verification - we don't have any control over this.
